From 6b83a36c089d02f8be010be68921aa1c30778652 Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Thu, 26 Mar 2015 18:19:58 +0800
Subject: [PATCH] MLK-10475-7 ARM: imx_v7_defconfig: Enable PRE driver support

This patch enables PRE driver support.

The new imx_v7_defconfig is generated in this way:
* make ARCH=arm imx_v7_defconfig
* make ARCH=arm menuconfig and manually choose to build in
  the PRE driver
* make ARCH=arm savedefconfig
* cp defconfig arch/arm/configs/imx_v7_defconfig

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
---
 arch/arm/configs/imx_v7_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/imx_v7_defconfig b/arch/arm/configs/imx_v7_defconfig
index b984956..4dbd0cb 100644
--- a/arch/arm/configs/imx_v7_defconfig
+++ b/arch/arm/configs/imx_v7_defconfig
@@ -323,6 +323,7 @@ CONFIG_MMC_SDHCI_PLTFM=y
 CONFIG_MMC_SDHCI_ESDHC_IMX=y
 CONFIG_MXC_IPU=y
 CONFIG_MXC_GPU_VIV=y
+CONFIG_MXC_IPU_V3_PRE=y
 CONFIG_MXC_MIPI_CSI2=y
 CONFIG_MXC_MLB150=m
 CONFIG_NEW_LEDS=y
-- 
1.8.0

